include/config/config_audio.h

```diff
@@ -22,6 +24,8 @@
 #define MAX_SIMULTANEOUS_NOTES_EMULATOR 40
 #define MAX_SIMULTANEOUS_NOTES_CONSOLE 24
 
-// Uses a much better implementation of reverb over vanilla's fake echo reverb. Great for caves or eerie levels, as well as just a better audio experience in general.
-// Reverb parameters can be configured in audio/synthesis.c to meet desired aesthetic/performance needs. Currently US/JP only. Hurts emulator and console performance.
-// #define BETTER_REVERB
+/** 
+ * Uses a much better implementation of reverb over vanilla's fake echo reverb. Great for caves or eerie levels, as well as just a better audio experience in general.
+ * Reverb parameters can be configured in audio/synthesis.c to meet desired aesthetic/performance needs. Currently US/JP only. Hurts emulator and console performance.
+ */
+// #define BETTER_REVERB
```

Comment:
needs a newline at the bottom 
```suggestion
// #define BETTER_REVERB

```